www/lib/ipfs.ts

```diff
@@ -14,7 +14,7 @@ export async function postJSONToIPFS(data: any, addDataTag: boolean = true) {
     };
   }
 
-  const result = await fetch('/api/uploadJson', {
+  const result = await fetch('https://rwtp.org/api/uploadJson', {
```

Comment:
doesn't hardcoding the origin make this harder to fork or test locally? it should at least be configurable in a single constant or using an environment variable

Reply:
Yes, but it's more of a pain without hardcoding than it is with. If you run it locally, you need to get an infura key. 

I could do an env variable